components/Article/Article.module.scss

```diff
@@ -42,7 +42,7 @@
   display: flex;
   align-items: center;
   justify-content: space-between;
-
+  margin-top: 0.5rem;
```

Comment:
@musojon1 add margin only if we are showing status. In places where we are not showing status it will change UI also (0.3rem) is enough